lib/get-ast.js

```diff
@@ -16,10 +16,19 @@ module.exports = (node) => {
             ...program,
             directives: [],
         },
-        comments,
+        comments: comments.map(transformComment),
         tokens,
     };
     
     return ast;
 };
 
+function transformComment(comment) {
```

Comment:
Let's move out this function into file `comments.js` with a name:

```suggestion
function convertProgramComments(comments) {
```

And there use `map`.